CONTRIBUTING.md

```diff
@@ -58,6 +58,44 @@ module.exports = function ModuleName(options,UI) {
 }
 ```
 
+Modules can be distributed independently via npm, yarn and cdn(for browser). `require` syntax should not be used if the module is intended to be used inside the browser.
+Any Independent Module Must follow this basic format
+
+```js
+function ModuleName(options,UI) {
+
+  var output;
+
+  function draw(input,callback) {
+
+    var output = function(input){
+      /* do something with the input */
+      return input;
+    }
+
+    this.output = output(input); // run the output and assign it to this.output
+    callback();
+  }
+
+  return {
+    options: options,
+    draw: draw,
+    output: output,
+    UI: UI
+  };
+}
+/* Info can be defined here or imported from a json file */
+var info = {
+      "name": "ModuleName",
+      "description": "",
+      "inputs": {
+        // inputs here
+      };
+      // var info = require('./info.json') This only works in node
```

Comment:
`In a module compiled with browserify or webpack, a require() can be used to load a standard info.json file.`